short-read-mngs/idseq_utils/idseq_utils/batch_run_helpers.py

```diff
@@ -1,77 +1,106 @@
-import time
-import random
-import boto3
 import json
-import requests
 import logging
+import os
+import random
+import re
+import requests
+import time
+from os import listdir
+from multiprocessing import Pool
+from subprocess import run
+from typing import Dict, List
+from urllib.parse import urlparse
+
+from idseq_utils.diamond_scatter import blastx_join
+from idseq_utils.minimap2_scatter import minimap2_merge
+
+import boto3
 from botocore.exceptions import ClientError
 
 log = logging.getLogger(__name__)
 
 MAX_CHUNKS_IN_FLIGHT = 10
+ALIGNMENT_WDL_VERSIONS: Dict[str, str] = {
+    "diamond": "v1.0.0",
+    "minimap2": "v1.0.0",
```

Comment:
Eventually it might be good to have these in some sort of parameter store where we can increment as we release new versions

Reply:
While this way is clunky I actually think hard-coding it is the best we can do right now. It is important that the version of the overall pipeline should include the versioning of these pipelines. So for example v6.11.0 of the main pipeline would correspond to v1.0.0 of the aligners. To enforce this we would need to release the aligners then explicitly upgrade them as a change to the main pipeline.